ibutsu_utils/xmerge.py

```diff
@@ -32,6 +37,8 @@ def merge_files(filenames):
 def main():
     args = parse_args()
     output_xml = merge_files(args.input)
+    if not output_xml:
+        return
```

Comment:
Do we also need a warning message here?

Reply:
Generally you're only going to hit that point if there are not files. I used to have a message there, but then I got both the message about the missing file and the message about no output, so I figured it was a little overboard.

Reply:
```
$ ibutsu-xmerge doesnotexist.xml
WARNING: doesnotexist.xml does not exist
WARNING: No files provided
```

Reply:
Aha! That makes sense.